src/main/java/jenkins/plugins/slack/workflow/SlackResponse.java

```diff
@@ -65,4 +65,20 @@ public boolean addReaction(String emojiName) {
 
         return slackService.addReaction(channelId, ts, emojiName);
     }
+
+    /**
+     * Remove an emoji reaction to the message that this `SlackResponse` points to.
+     *
+     * @param emojiName - name of the emoji (no colons), e.g. `thumbsup`
+     *
+     * @return boolean indicating whether the API request succeeded
+     */
+    @Whitelisted
+    public boolean removeReaction(String emojiName) {
+        if (slackService == null) {
+            return false;
+        }
+
+        return slackService.removeReaction(channelId, ts, emojiName);
```

Comment:
formatting is off

Reply:
Sorry but I don't saw where formating is broken ?
If I format code with my IDE (Intellij) severals change will be made, is there an editorconfig (or similar) or like to share options of format code ?

Reply:
Passing by your PR i see no problems with your formatting, @timja can we go with it?